rct229/rulesets/ashrae9012019/ruleset_functions/get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM_test.py

```diff
@@ -0,0 +1,153 @@
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM import (
+    get_fan_system_object_supply_return_exhaust_relief_total_kw_cfm,
+)
+from rct229.schema.config import ureg
+
+TEST_RMD = {
```

Comment:
You don't have to add units to the RMD.
Here is the example you can follow to quantify the rmd:
https://github.com/pnnl/ruleset-checking-tool/blob/develop/rct229/rulesets/ashrae9012019/ruleset_functions/get_building_scc_window_wall_ratios_dict_test.py#L139